### ISSUE TITLE
builds with Go 1.12 are broken

### ISSUE THREAD

**Issue by unclejack** [mholt/certmagic#47]
## What would you like to have changed?
The older versions don't work any more due to some dependencies being force pushed. The latest version doesn't work with Go 1.12.x. Projects which depend on certmagic can't be built with Go 1.12.
0.7.2 no longer works because https://github.com/labbsr0x/goh has been force pushed.

## Why is this feature a useful, necessary, and/or important addition to this project?
Builds should still work with the old stable Go version.

**Comment by mholt** [comment]
... I'm not sure what this has to do with certmagic, unfortunately; and there's not enough info here for me to do anything about it. :(

**Comment by mholt** [comment]
What is the error you're getting at build?

**Comment by unclejack** [comment]
With Go 1.12 and master:
```
/github.com/mholt/certmagic/client.go:143:8: ht.ForceAttemptHTTP2 undefined (type *http.Transport has no field or method ForceAttemptHTTP2)
```
With certmagic 0.7.2 and Go 1.12:
```
go: github.com/labbsr0x/goh@v0.0.0-20190417202808-8b16b4848295: unknown revision 8b16b4848295
```

**Comment by mholt** [comment]
Oh, right, you have to use Go 1.13 to build the latest versions of CertMagic, because it uses a feature only available in Go 1.13. If you can't upgrade to Go 1.13 (why not?), just use the previous version of CertMagic.

**Comment by unclejack** [comment]
I've taken a look at that dependency. The lego 3.0.2 release depends on that particular commit of the goh repo. That goh commit exists only on the Go module proxy now. Setting `GOPROXY="https://proxy.golang.org"` fixes the build issues for version 0.7.2 of certmagic.
